src/web/src/pages/index.tsx

```diff
@@ -1,13 +1,17 @@
+import { useState } from 'react';
 import SEO from '../components/SEO';
 import Banner from '../components/Banner';
 import Posts from '../components/Posts';
+import NavBar from '../components/NavBar';
 
 const Home = () => {
+  const [bannerIsVisible, setBannerVisibility] = useState<boolean>(true);
   return (
     <>
       <SEO pageTitle="Telescope" />
-      <Banner />
+      <Banner onVisibilityChange={(event) => setBannerVisibility(event)} />
```

Comment:
`onVisibilityChange={()=> setBannerVisibility(!bannerIsVisible)}`

What do you think?

Reply:
I tried this, it doesn't give the correct value somehow